README.md

```diff
@@ -55,6 +75,16 @@ A reference implementation of such a daemon is provided within the examples. The
 
 Currently all requests are initiated by wsfs daemon and responded by filesystem provider. This may change in future, e.g. when authentication is supported.
 
+## Similar Projects
+
+### Davfs2
+
+[davfs2](http://savannah.nongnu.org/projects/davfs2) is a Linux file system driver that allows to mount a [WebDAV](https://wiki.archlinux.org/index.php/WebDAV) resource. WebDAV is an extension to HTTP/1.1 that allows remote collaborative authoring of Web resources.
+
+Unlike fuse wsfs, davfs2 mounts a remote filesystem locally, that is provided by a WebDAV server. In contrast, fuse wsfs starts a server awaiting client connections to attach the remote file system.
```

Comment:
"Fuse WSFS", "fuse-wsfs" or "fuse wsfs"? Should use consistent naming across the whole document.

Reply:
see above